app/src/main/res/values/colors_v2.xml

```diff
@@ -32,4 +32,5 @@
 
   <color name="launcher_background">@color/white100</color>
   <color name="facility_search_query_highlight">@color/blue2</color>
+  <color name="status_bar">#0a2e52</color>
```

Comment:
Is this status bar's color not a standard color used across other screens?

Reply:
Nope, there's a separate color in Zeplin only for this: https://zpl.io/aR1dWmp